client/components/common/UpdateVersionModal/index.jsx

```diff
@@ -34,6 +42,21 @@ const UpdateVersionModal = ({
         setUpdatedDateAvailabilityText(value);
     };
 
+    const handleDateAvailabilityTextKeyPress = e => {
+        let input = e.target;
+        if ((e.charCode < 48 && e.charCode !== 45) || e.charCode > 57) {
```

Comment:
Maybe assign these codes to variables so it's easier to tell what specific keys we're looking for?

Reply:
Good point. Was hoping the following comment was enough but reading it back, this could be explained much better with variable names and expanding on that comment.